frontend/src/components/RecordsList.tsx

```diff
@@ -257,22 +261,20 @@ export default function RecordsList({
         </Modal>
       )}
 
-      {samplesQueryParam && (
+      {Object.keys(params).length !== 0 && (
         <AutoSizer>
           {({ height, width }) => (
             <Modal show={true} dialogClassName="modal-90w" onHide={handleClose}>
-              <Modal.Header closeButton>
-                <Modal.Title>{`Viewing ${samplesQueryParam}`}</Modal.Title>
-              </Modal.Header>
```

Comment:
Title was moved to be inside `SamplesList` so that the Patient Samples view can dynamically access `cmoPatientId`/`dmpPatientId` value and show it in its title.